### ISSUE TITLE
Menu items: Organizing the core items of Browsing

### ISSUE THREAD

**Issue by tesshucom** [tesshucom/jpsonic#2397]
Prerequisites: #2349

### Overview

Organize relationships between indexes, folders, artists, and albums. These are the most important items in Browsing. Determine the following points.

 - What kind of web view do you need?
 - What web menu items do you need?

<details>
<summary>Current situation</summary>

Web pages and UPnP currently have the following items. Several items of UPnP were implemented during the Subsonic/Airsonic era. Jpsonic has reimplemented it so that it can use UPnP paging communication. Also added some items. (Originally, this was not considered complete. Anyway, I hurriedly made something usable.) 

As a result, it has become easier to use and faster than before. However, **neither Web nor UPnP are sophisticated**.

![image](https://github.com/tesshucom/jpsonic/assets/27724847/4fa8d935-bbba-4167-a5fe-193182a1f053)

#### UPnP items

Fundamentally, we need to ensure that similar functionality is provided on the web and UPnP. However, it is impossible to realize UPnP App and Web App with exactly the same view. Rather, there is something like a realistic aptitude. By the time JavaScript started infinite scrolling, UPnP Apps would have loaded thousands of items and users will start doing fast swipes. 

 - Many UPnP Apps has a hierarchical/Tree type View, but the Web can do more than that.
 - In cases like simple listing of large amounts of data, view of native UPnP Apps are much faster than HTML or JavaScript in the browser. 

Therefore, they do not have to have exactly the same design. Currentry, It seems that UPnP does not need functional modification of Browsing item. However, **the settings page needs to be made simpler, with menu settings and options, and improved to make it easier to understand**.

#### Web pages

Three is enough.

 - Index (File Structure)
 - Index (ID3)
 - Albums (ID3)

However, we may need to be careful with the names. 

 - Index (File Structure) -> In the media server community, it may be often called "Folder View"?
 - Index (ID3) -> "Artsits" will be used worldwide
 - Albums (ID3) -> "Albums " will be used worldwide

Either way, the name will be modified so that the user can change it.

### The Design of  Web pages

We will need to modify, and create new ones.

### Artist

In Jpsonic's UPnP, Index is expressed as an independent layer. Even for the same Album, there may be differences in the display, for example, as shown below. If you're familiar with classical music, Folder View is almost a must-have.

 - **File Structure Indexes**
   - `A` **/** `Antonín Leopold Dvořák` **/** `Symphonic` **/** `Dvořák_ Symphonies #7-9 [Disc 2]` **/** `(Songs)`
 - **ID3 Indexes** : 
   - `V` **/** `Various Classical Artists` **/** `Dvořák_ Symphonies #7-9 [Disc 2]` **/** `(Songs)`

If we reproduce this on a web page, it will be difficult to use. It might be a good idea to add Index to the View in the Artist hierarchy as a "Jump to ..." feature.

This is not to say that this is a completely new idea. If anything, it's Origin. Subsonic Music Streamer is recognized as providing Folder View to most users. What is actually used is the File Structure Index. If you press and hold the scroll bar, you can refer to the Music Index. It's an excellent UI, but it's so sophisticated that no one notices its existence.　

Therefore, we can think of it as reimplementing something compatible  on web pages, with a "jump to" feature. 

### Album

I find this difficult to handle on the web. Therefore, use what is currently implemented. However, it would be nice to be able to switch to ID3. However, personally, it is a low priority for me. To be honest, I don't have a problem even if the web page doesn't have this. If anything, I think it would be more useful to delete it and use that time to create a Composer tree. However, since it has existed for a long time, it will be maintained.

</details>

#### Goal

Ther features relationship would be as follows. By changing the settings page, UPnP will be changed to an optional switching method. The current albums will be maintained (ID3 swith provided).

![image](https://github.com/tesshucom/jpsonic/assets/27724847/5078a8e1-c501-41a1-b8c4-f60aa0438f3a)
